README.md

```diff
@@ -1,15 +1,24 @@
 # danger-undercover
 
-A Danger plugin to use with Undercover gem.
+A [Danger](https://github.com/danger/danger) plugin to show [Undercover](https://github.com/grodowski/undercover) report.
 
 ## Installation
 
     $ gem install danger-undercover
 
 ## Usage
 
-    Methods and attributes from this plugin are available in
-    your `Dangerfile` under the `undercover` namespace.
+To use this gem all the instruction provided in [Undercover](https://github.com/grodowski/undercover) must be followed.
+Run the below command to output undercover report to a `txt` file which this plugin will use to geneate PR comments.
+To use it in a CI run this command before running the `Danger` so that the file is created beforehand.
+
+    $ undercover -c origin/master > coverage/undercover.txt 
```

Comment:
Yes, it's up to what each project needs. Added the master there as an example. Added more instruction in Readme for more clarification on running the commands. Changed in [1e51e90](https://github.com/nimblehq/danger-undercover/pull/1/commits/1e51e90b8f2c148b3ef419132aa9da2dd611080f)

Reply:
maybe we can change the `origin/master` to something like `origin/$current_branch` and explain the $current_branch @rafayet-monon

Reply:
@andyduong1920 it doesn't necessarily need to be a branch. It can be commit hash, branch name, or tag. I added in the readme of [1e51e90](https://github.com/nimblehq/danger-undercover/pull/1/commits/1e51e90b8f2c148b3ef419132aa9da2dd611080f)

> Use the `-c --compare ref` flag to specify a git ref (commit hash, branch name, tag) to compare against. 
**This is a recommended usage for CI/CD build environments**, as `undercover` will `exit 1` if there are any warnings.

so how about changing it to -

`$ undercover -c $compare_against > coverage/undercover.txt `

> Here `$compare_against` can be a commit hash, branch name, or tag. i.e. `origin/master`, `origin/development`

What do you think?

Reply:
Oh good to know that it accepts the `commit` too 👍

Reply:
@andyduong1920 Updated in [c73b2d2](https://github.com/nimblehq/danger-undercover/pull/1/commits/c73b2d2e8a12d4d227baa29cdcdfbaf8f70b4500)